index.html

```diff
@@ -3165,12 +3163,9 @@ <h3>Exploration Mechanisms</h3>
         </p>
        </section>
         <p>
-        The <a>Discovery</a> mechanisms defined in [[!WOT-DISCOVERY]] are not mandatory, but
+        The <a>Discovery</a> mechanisms defined in [[?WOT-DISCOVERY]] are not mandatory, but
         they are designed to meet the above requirements and their use will enhance 
         interoperability and usability.
-        <span class="rfc2119-assertion" id="arch-discovery-should-use-standard">Whenever
```

Comment:
note that this assertion was removed.  The idea that use of WoT discovery is optional is covered by the previous sentence.